examples/quick_template_score_calc.ipynb

```diff
@@ -528,7 +520,7 @@
    "source": [
     "temperature_score = TemperatureScore(\n",
     "    time_frames = [ETimeFrames.LONG],\n",
-    "    scopes=[EScope.S1S2],\n",
+    "    scopes=[EScope.S1S2, EScope.S3],\n",
```

Comment:
From slack char with Michael:

Are our benchmark files out of sync?  Using the benchmark file from your GitHub, I get KeyError: "[('Europe', 'Construction Buildings', <EScope.S3: 'S3'>)] not in index" when I run the notebook.  I’ll generate what I think is a correct and fulsome benchmark file.

Reply:
Maybe we use different files as inputs. I saw this error message, when I switched to one of xls in the folder. But if I run ITR_UI w/o arguments - the default file works w/o exceptions

Which input xls do you run?

Reply:
From slack chat with Michael:

I run with the 20220927 ITR Tool Sample Data.xls input (which has CBRE).

Reply:
Gotcha. I will fix my branch, when running with this file

Reply:
@MichaelTiemannOSC i can reproduce the exception `"[('Europe', 'Construction Buildings', <EScope.S3: 'S3'>)] not in index"`.
It's caused by the fact, that `benchmark_EI_OECM_PC.json`, and as a result, `benchmark_EI_OECM_S3.json`, don't have "Construction Buildings" benchmarks under S3 scope. I see "Construction Buildings" for S1S2, but not for S3.

Where do i get data, to fill the gaps for S3?

Reply:
I responded in email that this is an unresolved question, and for the moment treat S3 for buildings as zero.

Reply:
I added a workaround, to exclude "Construction buildings" sector from S3 scope